package-lock.json

```diff
@@ -17,7 +17,7 @@
                 "@deriv-com/analytics": "1.11.0",
                 "@deriv-com/quill-tokens": "^2.0.4",
                 "@deriv-com/quill-ui": "1.13.42",
-                "@deriv-com/translations": "1.3.5",
+                "@deriv-com/translations": "1.3.9",
```

Comment:
What’s the reason for the 1,211 lines of changes?

Reply:
@matin-deriv , lemme double check, I just completed the command as per our standard flow

Reply:
@matin-deriv , I checked it with Ako, it is expected.
thanks for you review 🙏